Demos/Run-BareMetal.CoolWorld.x86.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 cd ../bin
-./Mosa.Tool.Launcher.Console -autostart -oMax -output-asm -output-debug -output-hash -vmware-svga -include Include Mosa.BareMetal.CoolWorld.x86.dll
```

Comment:
Should be Console on Linux. 

The command line is customary on Linux.

Reply:
Sure it is, but some people prefer to not use it at all times. Even if people using MOSA are developers which means they'll be more easy with the command line, sometimes it's just nicer and easier to use GUI tools.

Reply:
And that's fine. The demo should fit what is customary on the platform. User can create their own scripts, if they want to non-customary usage.

Reply:
As a compromise, let's add both versions for Linux and Windows.